paper/utils.Rmd

```diff
@@ -0,0 +1,38 @@
+---
+title: Utils for the paper
```

Comment:
Why have a separate utils.Rmd vs. just including this code in the paper.Rmd? :thinking:

Reply:
The paper.md is not valid .Rmd (there were issues with the headers), so I decided to separate out the generation of the figures and tables.
I can rename this file to paper.Rmd, but I think this  is clear too.